### PR TITLE
updated celery and dependencies.

### PR DESCRIPTION
**Pull request by Maxyme**
This is for celery 4.1.0 and updated dependencies.

### CODE REVIEW

File: billiard/meta.yaml [hunk 1]
```diff
@@ -26,6 +27,4 @@ test:
 about:
   home: http://github.com/celery/billiard
   license: BSD 3-Clause
-  license_family: BSD
-  license_file: LICENSE.txt
-  summary: Python multiprocessing fork with improvements and bugfixes
+  summary: Python multiprocessing fork with improvements and bugfixes
```
Comment (Korijn) [inline billiard/meta.yaml]: missing newline

File: celery/meta.yaml [hunk 2]
```diff
@@ -1,50 +1,54 @@
...
-        - billiard >=3.5.0.2,<3.6.0
-        - kombu >=4.0.2,<5.0
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - pytz
+    - billiard >=3.5.0.3
+    - kombu >=4.1.0
```
Comment (Korijn) [inline celery/meta.yaml]: too broad

File: celery/meta.yaml [hunk 3]
```diff
@@ -1,50 +1,54 @@
...
+    - celery multi help
+    - celery shell --help
 
 about:
-    home: http://celeryproject.org
-    license: BSD License
-    summary: Distributed Task Queue
+  home: http://celeryproject.org
+  license: BSD License
+  summary: Distributed Task Queue
```
Comment (Korijn) [inline celery/meta.yaml]: missing newline

File: kombu/meta.yaml [hunk 4]
```diff
@@ -1,33 +1,34 @@
...
-        - setuptools
-    run:
-        - python
-        - amqp >=2.1.4,<3.0
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - amqp >=2.2.1
```
Comment (Korijn) [inline kombu/meta.yaml]: too broad

File: amqp/meta.yaml [hunk 5]
```diff
@@ -1,31 +1,30 @@
...
-        - setuptools
-    run:
-        - python
-        - vine >=1.1.3
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - vine >=1.1.4
```
Comment (Korijn) [inline amqp/meta.yaml]: too broad

File: billiard/meta.yaml (hunk 1 above)
Comment (Maxyme) [inline billiard/meta.yaml]: We have not done recipe linting in the past. If you really prefer a newline I can add it, some other recipes don't have it.

File: celery/meta.yaml (hunk 2 above)
Comment (Maxyme) [inline celery/meta.yaml]: This method of version pinning (i.e >= ) is to my knowledge the current standard. Limiting to a upper version does not guarantee full compatibility.

**Comment by Korijn** [comment]
OK, go for it

**Pull request merged**